byway/src/main/webapp/recommendations.js

```diff
@@ -30,8 +34,8 @@ const MAX_RECOMMENDATIONS = 1;
 // Holds regions google.maps.LatLng objects
 let regions = [];
 
-// holds markers as google.maps.Marker objects
-let markers = [];
+// Boolean to determine when to toggle markers
+let isHiding = true;
```

Comment:
Nitt: would call this something slightly more specific and related to what is actually being Hidden. E.g. "waypoinsAreHidden" or "areMarkersHidden"

Reply:
Fixed to be more specific. Also, changed some of the logic to track the true state of them (i.e. aren't initially hidden, only when toggled odd number of times - off by one state)